src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -118,6 +121,15 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngraph::F
     // before CommonOptimization pipeline execution
     manager.register_pass<ngraph::pass::MOCTransformations>(true, false);
 
+    // Marks constant folded shape values to be kept in FP32.
+    // When dynamism on GPU will be turned on
+    // need to keep the whole shape subgraph not only constants
+    manager.register_pass<ov::pass::MarkPrecisionSensitiveSubgraphs>();
```

Comment:
Is it needed only for GPU? Maybe disable it by default?

Reply:
in the version all markings and aligning are moved inside `ConvertCompressedToMixedPrecision` which is disabled by default